packages/schema/bind/src/bindings/index.ts

```diff
@@ -21,6 +21,10 @@ export function getGenerateBindingFn(
       );
     case "wrap-go":
       return Golang.Wasm.generateBinding;
+    case "wrap-ts":
+      return WrapBindgen.getGenerateBindingFn(
+        "wrap://http/http.wrappers.dev/u/test/wrap-typescript-wrap-abi-bindgen"
```

Comment:
This can be changed to `wrapscan.io/polywrap/wrap-typescript-abi-bindgen@1`:
https://github.com/polywrap/wrap-abi-bindgen/pull/103